### PR TITLE
Fix kubetest2-aks pipeline for release branches

### PR DESCRIPTION
**Pull request by lzhecheng**
Signed-off-by: Zhecheng Li <zhechengli@microsoft.com>

#### What type of PR is this?
/kind testing

#### What this PR does / why we need it:
Fix kubetest2-aks pipeline for release branches
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: .pipelines/ccm-image.yml [hunk 1]
```diff
@@ -9,9 +9,15 @@ steps:
       export GOPATH="/home/vsts/go"
       export PATH="${PATH}:${GOPATH}/bin"
 
+      if [[ ! -d kubetest2-aks ]]; then
+        git clone https://github.com/kubernetes-sigs/cloud-provider-azure.git
+        cp -r cloud-provider-azure/kubetest2-aks .
+        rm -rf cloud-provider-azure
+      fi
       pushd kubetest2-aks
```
Comment (MartinForReal) [inline .pipelines/ccm-image.yml:17]: should remove this line as well?

File: .pipelines/cnm-image.yml [hunk 2]
```diff
@@ -9,9 +9,15 @@ steps:
       export GOPATH="/home/vsts/go"
       export PATH="${PATH}:${GOPATH}/bin"
 
+      if [[ ! -d kubetest2-aks ]]; then
+        git clone https://github.com/kubernetes-sigs/cloud-provider-azure.git
+        cp -r cloud-provider-azure/kubetest2-aks .
+        rm -rf cloud-provider-azure
+      fi
       pushd kubetest2-aks
```
Comment (MartinForReal) [inline .pipelines/cnm-image.yml:17]: the same as the previous one.

**Comment by lzhecheng** [comment]
> So to fix the pipeline, we don't need to change the format of [cluster template](https://github.com/kubernetes-sigs/cloud-provider-azure/blob/06c649ea4095527000e0cc50e7f64d6771e8222b/kubetest2-aks/cluster-templates/autoscaling.json#L1-L0) to match the [API model](https://github.com/kubernetes-sigs/cloud-provider-azure/blob/40a153c960e8919c3232bddc3ed5459182fd852a/kubetest2-aks/deployer/up.go#L144)?

Manually tested and it worked but the fix for that problem is #2708

**Pull request merged**